Library/Homebrew/download_strategy.rb

```diff
@@ -830,6 +838,10 @@ def stage
     safe_system(*args)
   end
 
+  def source_modified_time
+    Time.parse Utils.popen_read("fossil", "info", "tip", "-R", cached_location.to_s)[/^parent: *\h* (.*)$/, 1]
```

Comment:
> It's something wrong with me.

There's nothing wrong with you! That's just human and happens to all of us all the time. 😸